epoch/api/contract_api_usage.md

```diff
@@ -18,7 +18,7 @@ For an up to date list of all HTTP API endpoints and their arguments see
 An epoch node provides some utility functions to help you create contract transactions.
 
 
-### Contract Compile 
+### Contract Compile
```

Comment:
I see updates to this file deliver https://www.pivotaltracker.com/story/show/157839891